paperqa/readers.py

```diff
@@ -39,7 +41,25 @@ def parse_pdf_to_pages(
                     f" {file.page_count} for the PDF at path {path}, likely this PDF"
                     " file is corrupt."
                 ) from exc
-            text = page.get_text("text", sort=True)
+
+            if os.environ.get("PQA_USE_BLOCK_PARSING", "0").lower() in ENABLED_LOOKUP:
```

Comment:
@markenki we are making this opt-in for now to not break people (including ourselves at FutureHouse). If upon our next baseline this doesn't harm performance, we'll make this the default behavior